Gemfile

```diff
@@ -5,8 +5,8 @@ gem 'jira-ruby', :group => :development
 
 group :test do
   gem 'rspec'
-  gem 'beaker', '~>1.17.0'
+  gem 'beaker', '~> 2.2'
   if ENV['GEM_SOURCE'] =~ /rubygems\.delivery\.puppetlabs\.net/
-    gem 'sqa-utils'
+    gem 'sqa-utils', '> 0.11'
```

Comment:
Hmm, I think I intended for this to be `~> 0.11` which I believe would prevent major version updates while still giving us minor and patch version updates.